config/tests/guest/libvirt/upstream.cfg

```diff
@@ -246,5 +239,87 @@ variants:
+                    - memory:
+                        only avocado_guest.memhotplug,avocado_guest.eatmemory,avocado_guest.memintegrity
+                    - generic:
+                        only smallpages
+                        only avocado_guest.ltp,avocado_guest.kselftest
+                    - cpu:
+                        only smallpages
+                        vcpu_cores = 4
+                        vcpu_threads = 8
+                        vcpu_sockets =  1
+                        vcpu_maxcpus = 32
+                        only avocado_guest.ebizzy,avocado_guest.ppc64_cpu_test
+                    - io:
+                        vcpu_cores = 16
+                        vcpu_threads = 2
+                        vcpu_sockets =  1
+                        vcpu_maxcpus = 32
+                        only avocado_guest.fiotest
+
+            - guestdump:
+                only virsh.dump.positive_test.non_acl.live_dump
+            - diskhotplug:
+                at_dt_disk_device_target = sdb
+                only virsh.attach_detach_disk_matrix..at_option_live
+                no dt_option_live_config,dt_option_config,dt_option_default,dt_option_live_config,at_okay_dt_error,at_error_dt_error,pre_vm_state_shutoff,pre_vm_state_paused,pre_vm_state_transient,dt_option_current
+            - memoryhotplug:
+                only libvirt_mem.positive_test.memory.hot.unplug.max_slots.with_rand_reboot
+            - vcpuhotplug:
+                start_vm = yes
+                test_itr = 2
+                mem = 81920
+                vcpu_max_timeout = 480
+                vcpu_sockets =  1
+                vcpu_maxcpus = 64
+                topology_correction = "no"
+                variants:
+                    - 1thread:
+                        no hugepages
+                        vcpu_cores = 64
+                        smp = 1
+                        vcpu_threads = 1
+                        vcpu_current_num = 1
+                        vcpu_plug_num = 64
+                        vcpu_unplug_num = 1
+                        vcpu_max_num = 64
+                        only libvirt_vcpu_plug_unplug..with_maxvcpu
+                    - 2threads:
+                        no hugepages
+                        smp = 2
+                        vcpu_cores = 32
+                        vcpu_threads = 2
+                        vcpu_current_num = 2
+                        vcpu_plug_num = 64
+                        vcpu_unplug_num = 2
+                        vcpu_max_num = 64
+                        only libvirt_vcpu_plug_unplug..with_maxvcpu
+                    - 4threads:
+                        smp = 4
+                        vcpu_cores = 16
+                        vcpu_threads = 4
+                        vcpu_current_num = 4
+                        vcpu_plug_num = 64
+                        vcpu_unplug_num = 4
+                        vcpu_max_num = 64
+                        only libvirt_vcpu_plug_unplug..with_maxvcpu
```

Comment:
Is this the only variant that will use hugepages as backend?

We could name it as `4threads_hugepages` to differentiate it from other variants that do not use hugepages.

Reply:
that will be displayed in the test name like this, 
vm.vcpuhotplug.4threads.qemu.**hugepages**.qcow2.virtio_scsi.virtio_net.Guest.libvirt_vcpu_plug_unplug.positive_test.vcpu_set.live.with_maxvcpu